pkg/resources/kafka/configmap.go

```diff
@@ -346,6 +346,44 @@ func generateListenerSSLConfig(config *properties.Properties, name string, sslCl
 	}
 }
 
+// mergeSuperUsersPropertyValue merges the target and source super.users property value, and returns it as string.
+// It returns empty string when there were no updates or any  of the super.users property value was empty.
```

Comment:
nit - unintentional extra space in comment

```suggestion
// It returns empty string when there were no updates or any of the super.users property value was empty.
```

Reply:
You don't have to fix it now - I can fix it in my refactoring PR https://github.com/banzaicloud/koperator/pull/917 after yours is merged

Reply:
Thank you! I think it is better to fix it here.